### ISSUE TITLE
PKAs get higher cooldown if you carry an extra one in backpack

### ISSUE THREAD

**Issue by rasepretep** [tgstation/tgstation#70567]
I fully understand that the reason why PKAs have double cooldown on dual wielding them is to nerf the otherwise stupidly OP damage output, that's always been the case. I would even understand if it also applied to PKAs that are in belt or suit slot, because they remain charged and ready for use.

However, I really think PKAs placed inside backpacks should be exempt from contributing to the cooldown increase, for the following reasons, and that this is in fact either a bug or oversight:

1) PKAs in backpacks lose their charge / don't charge up. So swapping a number of PKAs between hand and backpack wouldn't help in any way, you'd never pull out a fully charged PKA to chain shots.
2) Having an extra PKA with different mods in your backpack doesn't provide much benefit to be gamebreaking. At most you can switch mining / damage / pvp PKAs without having to crowbar and re-mod it. A traitor miner could have one PKA for mining and one for station. Given how I've never seen anyone actually use the syndicate pressure mods, I doubt it'd change much.
3) It simply doesn't make sense that dragging a PKA behind you is fine, but having it in your backpack is not.

## Round ID:
Any recent round.

## Reproduction:
Put one PKA in backpack, hold the other one and fire fire it. You'll see the cooldown has almost doubled.

**Comment by LemonInTheDark** [comment]
lmao

**Comment by timothymtorres** [comment]
Relevant code:

https://github.com/tgstation/tgstation/blob/74418bad5494f05250d7dea37ff1e71923cfe81c/code/modules/projectiles/guns/energy/recharge.dm#L56-L71

**Comment by timothymtorres** [comment]
I wonder if this is intentional so that you can't just swap PKAs out of your backpack so you can avoid the cooldown?  Is it possible to hotkey items to and from backpack?

**Comment by rasepretep** [comment]
> I wonder if this is intentional so that you can't just swap PKAs out of your backpack so you can avoid the cooldown? Is it possible to hotkey items to and from backpack?

As I said, PKAs already lose charge / don't charge up in backpacks. Switching multiple between hand and backpack would achieve nothing, since every one you'd pull out wout still take the usual 3 seconds to charge, making it equal to having just one.

**Comment by optimumtact** [comment]
This isn't a bug, the original code is quite deliberate in intention and it was to stop people swapping out PKA's

even though that doens't work any more it still isn't a bug, it's a design oversight.

**Comment by necromanceranne** [comment]
You can wield two of them at once and get double the damage output. It's not just PKA juggling, it's to limit your dual-wielding power. Higher burst, lower cooldown.

**Comment by rasepretep** [comment]
> This isn't a bug, the original code is quite deliberate in intention and it was to stop people swapping out PKA's
> 
> even though that doens't work any more it still isn't a bug, it's a design oversight.

Agreed that it's an oversight. Or rather, somewhat makeshift solution to prevent swapping out PKAs to rapid chain shots, which was the initial problem.

I'd much prefer if PKAs would not interfere with each other unless both held in hands, but also that they would only charge up / keep charge while held in hand. That lets people keep a different one on them, but provides zero benefit for swapping them other than using what's better for current situation.

And on the subject of benefits, there really isn't that many. I've never seen the traitor pressure mods be used, so at best I can assume if someone does keep a spare, it'd be to have one with AOE for mining and one for killing fauna, or with max range to protect themselves better if needed.
